stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -133,11 +132,10 @@ public enum _RuntimeConfig {
 }
 
 private func configureRuntimeFromEnvironment() {
-  if let value = getenv("SWIFT_TENSORFLOW_ENABLE_DEBUG_LOGGING") {
-    if String(cString: value).lowercased() == "true" {
+  if let value = getenv("SWIFT_TENSORFLOW_ENABLE_DEBUG_LOGGING"),
+    String(cString: value).lowercased() == "true" {
```

Comment:
Why the chaining here?  None of the other cases do that.  The rest LGTM

Reply:
I figured the chaining was better than an extra nested `if`.  Did the same below on `160`.  Or maybe not understanding you correctly @compnerd

Reply:
Oh, I see, the folded bit had confused me.  This seems reasonable.